build.groovy

```diff
@@ -107,11 +114,7 @@ def initializeBuildProcess(String[] args) {
 	// verify/create build data sets required by each language script
 	if (props.languagePropertyQualifiers) {
 		props.languagePropertyQualifiers.trim().split(',').each { lang ->
-			if (props."${lang}_srcDatasets")
-				createDatasets(props."${lang}_srcDatasets".split(','), props."${lang}_srcOptions")
-		
-			if (props."${lang}_loadDatasets")
-				createDatasets(props."${lang}_loadDatasets".split(','), props."${lang}_loadOptions")
+		        buildUtils.getLanguagePropertyQualifiers(lang)
```

Comment:
While this is invoked during the initializationPhase of the build, it will call the getLanguagePropertyQualifiers for all languages defined. However the method getLanguagePropertyQualifiers will also create the datasets.

This means, when performing a UserBuild on a cobol file, it will generate all datasets for all other languages as well. This is what we wanted to avoid per #16